src/Nethermind/Nethermind.Consensus/Validators/TxValidator.cs

```diff
@@ -79,6 +82,7 @@ private bool ValidateChainId(Transaction transaction)
                     return true;
                 case TxType.AccessList:
                 case TxType.EIP1559:
+                case TxType.Blob:
```

Comment:
should this be just `default` ? Or just have if/else? I don't think we expect this to ever change for new TxTypes.